controllers/controllers.js

```diff
@@ -38,3 +38,18 @@ exports.getArticleIdComments = (realRequest,realResponse,next) =>{
         next(err)
     })
 }
+exports.postCommentsToArticle = (realRequest,realResponse,next) =>{
+    const {params} = realRequest
+    const promises = [fetchArticleById(params.article_id)]
+    if (params.article_id){
+        promises.push(addCommentsToArticle(realRequest.params.article_id,realRequest.body))
+    }
+    Promise.all(promises)        
```

Comment:
It shouldn't be necessary to check the article exists here, as article_id is a foreign key, PSQL will throw an error if you try to post to a non existent article :)